src/marker.rs

```diff
@@ -352,7 +352,7 @@ pub use nightly::Ungil;
 /// # Releasing and freeing memory
 ///
 /// The [`Python`] type can be used to create references to variables owned by the Python
-/// interpreter, using functions such as [`Python::eval`] and [`PyModule::import`]. These
+/// interpreter, using functions such as [`Python::eval`] and ``PyModule::import``. These
```

Comment:
What's up with the double ticks here?

(I guess the whole docs section here deserves a rewrite to represent the `Bound` API, but that's probably a bit out of scope here.)

Reply:
Sigh, I'm used to double-tickets from restructured text. Will fix!